barretenberg/cpp/src/barretenberg/constants.hpp

```diff
@@ -11,7 +11,7 @@ static constexpr uint32_t CONST_PROOF_SIZE_LOG_N = 28;
 // circuits being folded.
 static constexpr uint32_t CONST_PG_LOG_N = 20;
 
-static constexpr uint32_t CONST_ECCVM_LOG_N = 16;
+static constexpr uint32_t CONST_ECCVM_LOG_N = 15;
```

Comment:
You should prob tag this with e2e_prover or otherwise check the sizes manually in master to be sure this won't break the e2e test.